containers/Connector/Connector.tsx

```diff
@@ -104,25 +103,32 @@ const useConnector = () => {
 				wallet: async (wallet: OnboardWallet) => {
 					if (wallet.provider) {
 						const provider = loadProvider({ provider: wallet.provider });
-						const signer = provider.getSigner();
 						const network = await provider.getNetwork();
 						const networkId = network.chainId as NetworkId;
-						const useOvm = getIsOVM(networkId);
-
-						synthetix.setContractSettings({
-							networkId,
-							provider,
-							signer,
-							useOvm,
-						});
-						setProvider(provider);
-						setSigner(provider.getSigner());
-						setNetwork({
-							id: networkId,
-							name: network.name as NetworkName,
-							useOvm,
-						});
-						setSelectedWallet(wallet.name);
+						const networkName =
```

Comment:
What's the difference here between `network.name` and `networkName`?

Reply:
`network.name` for ovm provider is broken, is returning `unknown`..